.github/workflows/install-dependencies.sh

```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+#!/bin/bash
```

Comment:
Issue: It looks like `#!/bin/bash` needs to be at the very top of the file (according to [this StackOverflow post](https://stackoverflow.com/a/8968514)).

> when it is the first two bytes of an executable

This applies to all other `.sh` we've modified in this pull-request.

Reply:
thanks for the resource! I'll change them!